cpp/neighbors_module/neighbors_module.cpp

```diff
@@ -5,6 +5,11 @@
 extern "C" int mgp_init_module(struct mgp_module *module, struct mgp_memory *memory) {
   try {
     mgp::memory = memory;
+    AddProcedure(Neighbors::AtHop, Neighbors::kProcedureAtHop, mgp::ProcedureType::Read,
+                 {mgp::Parameter(Neighbors::kArgumentsNode, mgp::Type::Node),
+                  mgp::Parameter(Neighbors::kArgumentsRelType, mgp::Type::String),
+                  mgp::Parameter(Neighbors::kArgumentsDistance, mgp::Type::Int)},
+                 {mgp::Return(Neighbors::kReturnAtHop, mgp::Type::Node)}, module, memory);
```

Comment:
In APOC, apoc.neighbors.athop can take a relationship filter (`rel_types`) that accepts multiple types of relationships ([docs](https://neo4j.com/labs/apoc/4.3/graph-querying/neighborhood/#neighborhood-search-rel-filters)).
As there are already several PRs for functions that use relationship filters, e.g. #300 and #301, I suggest changing the second argument of this function so that it works the same way as in those PRs (i.e. as a list of strings).